### ISSUE TITLE
Colors as hash keys incorrectly detected as color keywords

### ISSUE THREAD

**Issue by Dorian** [causes/scss-lint#398]
It is quite easy to reproduce:

**example.scss**:

```
$colors: (
  green: #abcabc
);
```

```
$ scss-lint example.scss
example.scss:2 [W] ColorKeyword: Color `green` should be written in hexadecimal form as `#008000`
```

**Comment by sds** [comment]
Duplicate of #175. See comment in https://github.com/causes/scss-lint/issues/175#issuecomment-45644321

**Comment by Dorian** [comment]
Thanks, I may go for the quotes solution
